src/server/common/model/answer/answer-model.test.js

```diff
@@ -0,0 +1,69 @@
+import Joi from 'joi'
+import { AnswerModel, validateAgainstSchema } from './answer-model.js'
+
+describe('validateAgainstSchema', () => {
+  let validationSchema
+
+  beforeEach(() => {
+    validationSchema = Joi.object({
```

Comment:
schemas don't get mutated by being validated (may even be immutable?), so you can do: 

```javascript
const validationSchema = Joi.object({
  addressLine1: Joi.string().required(),
  postcode: Joi.string().required()
})
```

and drop the `beforeEach` block